package cloudutils4j.exceptions.s3.invalid.localpath;

public class EmptyDestinationLocalPathException extends RuntimeException {
    public EmptyDestinationLocalPathException(String message) { super(message); }
}
